examples/binding_pockets/README.md

```diff
@@ -0,0 +1,6 @@
+# Dataset Description
```

Comment:
The `binding_pockets` example is very experimental, but it's based on PDBBind I believe (binding pockets were extracted from PDBBind and used to train model). Details for PDBBind should be in https://arxiv.org/abs/1703.00564